apisix/discovery/nacos/init.lua

```diff
@@ -405,12 +434,17 @@ function _M.init_worker()
         return
     end
 
-    default_weight = local_conf.discovery.nacos.weight
-    log.info('default_weight:', default_weight)
+    --default_weight = local_conf.discovery.nacos.weight
```

Comment:
why change these?

Reply:
Because `default_weight` in different nacos may different